blue_moveit_config/launch/move_group.launch

```diff
@@ -32,7 +32,7 @@
   <!-- Trajectory Execution Functionality -->
   <include ns="move_group" file="$(find blue_moveit_config)/launch/trajectory_execution.launch.xml" if="$(arg allow_trajectory_execution)">
     <arg name="moveit_manage_controllers" value="true" />
-    <arg name="moveit_controller_manager" value="$(arg blue_side)" unless="$(arg fake_execution)"/>
+    <arg name="moveit_controller_manager" value="blue_full" unless="$(arg fake_execution)"/>
```

Comment:
Why is this hardcoded to `blue_full`? We pass in a `blue_side` argument, but it seems like it's basically ignored?

Reply:
Currently, the ```blue_side``` argument will only load in the specified moveit_sensor_manager and srdf. A move group for the full arm will be attempted. This was left over from when we had additional files which specified for each side. We can

1. Remove the reference to ```blue_side``` here
or 
2.  Add back in the files that use the ```blue_side``` argument.